_docs_operate/configuration.md

```diff
@@ -595,39 +595,39 @@ This module is deprecated in favor of the [Message Broker Publisher](#messagebro
 
   The server under the URL must respond to the request with a status code between 200 and 299. Otherwise the Connector will log a warning.
 
-  Additionally, you can configure an `authenticationProvider` for a target. The authentication provider is used to authenticate the request to the webhook. The available authentication providers are: `OAuth2` and `ApiKey`.
+  Additionally, you can configure an `authentication` for a target. The authentication is used to authenticate the request to the webhook. The available authentication methods are: `OAuth2` and `ApiKey`.
 
   **OAuth2**
 
-  The OAuth2 authentication provider is used to authenticate the request to the webhook using OAuth2. The Connector will send a bearer token as part of the request in its Authentication header. To configure the OAuth2 authentication provider, you need to provide the following parameters:
+  The OAuth2 authentication method is used to authenticate the request to the webhook using the client credentials flow of OAuth2. The Connector will send a bearer token as part of the request in its Authentication header. The OAuth2 authentication is configured using the following parameters:
 
   - **type** `"OAuth2", required`
 
-    The type of the authentication provider.
+    The type of the authentication method.
 
   - **accessTokenUrl** `string, required`
 
     The URL to get the access token from. This URL must be reachable from the Connector.
 
   - **clientId** `string, required`
 
-    The client id of the OAuth2 application.
+    The [client id](https://www.rfc-editor.org/rfc/rfc6749#section-3.2.1) of the access request.
 
   - **clientSecret** `string, required`
 
-    The client secret of the OAuth2 application.
+    The [client secret](https://www.rfc-editor.org/rfc/rfc6749#section-3.2.1) of the access request.
```

Comment:
```suggestion
    The [client secret](https://www.rfc-editor.org/rfc/rfc6749#section-3.2.1) used to authenticate to the access token server.
```

Reply:
lol, that was your suggestion :D

Reply:
No it wasn't. It was my suggestion for the scope. But you took it for every property.